docs/reference/modules/network.asciidoc

```diff
@@ -35,11 +34,23 @@ in the following table:
 
 |`_local:ipv6_` |Will be resolved to loopback IPv6 addresses
 
-|`_non_loopback_` |Addresses of the first non loopback interface
+|`_site_` |Will be resolved to site-local addresses ("private network")
```

Comment:
I was worried about the table getting too big. if we want to refactor the documentation, then I think examples might be a better place, I figure site-local is standard terminology, and "private network" is for the real morons, and examples should go elsewhere?